website/docs/dev_guide/high_level/conditions.md

```diff
@@ -58,16 +58,16 @@ E.g., the following condition is valid:
 var condition = Condition<EntityClass>.On(x => x.TopLvlProperty.NestedList[3].MoreNestedProperty).EqualsTo(10);
 ```
 
-### Comparison with other attributes
+### Comparison with other properties
 
-The majority of DDB condition operations support comparison with other attributes instead of outside value.
+The majority of DynamoDB condition operations support comparison with other properties instead of an explicit value.
```

Comment:
I changed it to "property" because a few lines down the text read:

> Some operations like `Between` accepts even combination of value and another **property**

It felt like "attribute" and "property" were being used interchangeably in this section, so I was trying to unify on a single term. Should all of these say "attribute"? I can see your point about using the term that is appropriate for the layer, dynamo vs C#.

Reply:
I see. I guess the best way would be to say `DDB attribute` everywhere except code examples. I feel that these should remain `entity.SomeProperty` or something like this. What do you think about it?